runtime/bcutil/ConstantPoolMap.cpp

```diff
@@ -105,7 +105,7 @@ ConstantPoolMap::computeConstantPoolMapAndSizes()
 		J9CPTYPE_UNUSED, /* 14 */
 		J9CPTYPE_METHODHANDLE, /* CFR_CONSTANT_MethodHandle */
 		J9CPTYPE_METHOD_TYPE, /* CFR_CONSTANT_MethodType */
-		J9CPTYPE_UNUSED, /* 17 */
+		J9CPTYPE_CONSTANT_DYNAMIC, /* CFR_CONSTANT_Dynamic */
```

Comment:
Should this be 17 or 18?  If it's meant to be 18 it needs an UNUSED to fill the 17th slot

Reply:
Constant Dynamic is 17